youtube_dl/extractor/rai.py

```diff
@@ -410,6 +439,11 @@ def _real_extract(self, url):
                 ''' % self._UUID_RE,
                 webpage, 'content item id', default=None, group='id')
 
+        if not content_item_id:
+            content_item_id = self._search_regex(
+                r'/ContentItem-(?P<id>%s)\.html\?iframe' % self._UUID_RE,
+                webpage, 'content item id', default=None, group='id')
+
```

Comment:
Do not mix unrelated changes in single PR.

Reply:
here: #27673